docs/latest/getting-started/custom-handlers.md

```diff
@@ -66,5 +66,5 @@ export const handler: Handlers = {
 ```
 
 Handlers can do much more, including fetching data from a database or external
-API and passing it to their route. See [Data fetching](/docs/concepts/) for more
+API and passing it to their route. See [Data fetching](/docs/concepts/data-fetching) for more
 details.
```

Comment:
Looks like we removed that page completely in https://github.com/denoland/fresh/pull/2107 . Let's get rid of that "See ..." sentence.

```suggestion
API and passing it to their route.
```

Reply:
Thanks for pointing that out!